packages/hooks/src/Context/global-context.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { createContext, ReactNode } from 'react';
+import { useLocalStorage } from 'usehooks-ts';
+
+import { useSubscription } from '@deriv/api';
+
+type TExchangeRatesContext = {
+    handleSubscription: (base_currency: string, target_currency: string) => void;
+    exchange_rates: TRate;
+    rest: Omit<typeof useSubscription, 'subscribe' | 'data'>;
+};
+
+type TRate = Record<string, Record<string, number | number>>;
```

Comment:
Isn't the second number redundant?
```suggestion
type TRate = Record<string, Record<string, number>>;
```

Reply:
you're right. removed it. good catch